themes/mongodb/src/navbar/navbar-dropdown.js

```diff
@@ -91,10 +91,10 @@ class NavbarDropdown extends preact.Component {
                                             <a href="https://docs.atlas.mongodb.com/">MongoDB Atlas</a>
                                         </li>
                                         <li className="submenu__item">
-                                            <a href="https://docs.atlas.mongodb.com/atlas-search/">MongoDB Atlas Search</a>
+                                            <a href="https://docs.mongodb.com/datalake/">MongoDB Atlas Data Lake</a>
                                         </li>
                                         <li className="submenu__item">
-                                            <a href="https://docs.mongodb.com/datalake/">MongoDB Atlas Data Lake</a>
+                                            <a href="https://docs.atlas.mongodb.com/atlas-search/">MongoDB Atlas Search</a>
```

Comment:
Switched these to be in alphabetical order

Reply:
That seems imminently reasonable.